### ISSUE TITLE
i18n gives linting error for material-icons text

### ISSUE THREAD

**Issue by maplion** [angular-eslint/angular-eslint#603]
I want to turn on the linter to check for i18n text that has not been translated and have it give me a warning (and eventually set it to error), but the problem is that it considers material-icons text untranslated text.  I can make the warning go away by adding an i18n tag to the material-icons text, but that breaks the icon.

I think the correct behavior would be to ignore linting on tags that are of class "material-icons" or there should be a configurable way to ignore html tags by a chosen class, such as material-icons [so I'm not sure if this is a bug that morphs into a feature request].

Example rules for html files:

```JSON
{
  "files": [
    "*.html"
  ],
  "extends": [
    "plugin:@angular-eslint/template/recommended"
  ],
  "rules": {
    "@angular-eslint/template/i18n": [
      "warn",
      {
        "checkId": true,
        "checkText": true,
        "checkAttributes": false
      }
    ]
  }
}

```
Example material-icon that causes a linting error:

```TS
<i class="material-icons">assignment</i>
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/eslint-plugin`    | `4.26.1` |
| `@typescript-eslint/parser`        | `4.26.1` |
| `ESLint`                           | `7.28.0` |
| `node`                             | `14.15.5` |

```sh
Angular CLI: 12.0.4
Node: 14.15.5
Package Manager: npm 7.17.0
OS: win32 x64

Angular: 12.0.4
... animations, cdk, cli, common, compiler, compiler-cli, core
... forms, language-service, localize, material
... material-moment-adapter, platform-browser
... platform-browser-dynamic, router, service-worker

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1200.4
@angular-devkit/build-angular   12.0.4
@angular-devkit/core            12.0.4
@angular-devkit/schematics      12.0.4
@schematics/angular             12.0.4
rxjs                            6.5.5
typescript                      4.2.4
```

- [ x ] I have tried restarting my IDE and the issue persists.
- [ x ] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by rafaelss95** [comment]
If you want to ignore texts within specific tags, you could use `ignoreTags` option for that, e.g.:

```json
{
  "rules": {
    "@angular-eslint/template/i18n": [
      "warn",
      {
        "checkId": true,
        "checkText": true,
        "checkAttributes": false,
        "ignoreTags": ["i"]
      }
    ]
  }
}
```

Let me know if it solves your issue.

**Comment by maplion** [comment]
@rafaelss95 Exactly what I needed, thank you!
